.github/actions/prepare/action.yml

```diff
@@ -11,7 +11,8 @@ inputs:
 runs:
   using: composite
   steps:
-    - uses: actions/setup-node@v3
+    - name: 'Use Node.js 18.x'
```

Comment:
It's not always 18.x, it's `inputs.node-version` but could you remove these so that it's less to update whenever we change the version?

Reply:
Fixed so now it's done using a single variable.